lib/common.sh

```diff
@@ -496,12 +496,8 @@ list_nodes() {
 iterate(){
   local RUNS=0
   local COMMAND="$*"
-  local TMP_RET TMP_RET_CODE
-  TMP_RET="$(${COMMAND})"
-  TMP_RET_CODE="$?"
 
-  until [[ "${TMP_RET_CODE}" = 0 ]] || [[ "${SKIP_RETRIES}" = true ]]
-  do
+  until "${COMMAND}" || [[ "${SKIP_RETRIES}" = true ]]; do
```

Comment:
```suggestion
  until "$@" || [[ "${SKIP_RETRIES}" = true ]]; do
```
$* makes it a single string, which probably the subshell expansion "rescued". Use "$@" directly.